radar/tools/yarn.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Yarn"
```

Comment:
we have to specify the `yarn version` 

`yarn 4` (the project linked in the description of this entry) is a totally different package manager from `yarn 1.x` (that is used on `io-app`, `io-backend`, `io-functions-app` ...)

in my opinion we should split this submission in two separate entries:
- `yarn 1.x, hold` 
- `yarn 4.x, adopt`

and, since `React Native` is not compatible with `Plug'n play` (the default mode since `yarn 2.x`) we can talk about the opportunity to use `yarn` for React Native over `npm`

Reply:
Sounds good to me, thx.

Reply:
I should convert this to the `yarn 1.x, hold`. I can add also a new PR for `yarn 4.x, adopt`: @lucacavallaro where are we using `4.x` version of `yarn`?

Reply:
`yarn 3.x`: `io-sign`, `io-wallet`, `io-link`, `io-std`. 3.x uses the same api as `4.0`

imho you can add here both entries, the title can be `Yarn Classic` (1.0) and `Yarn`

Reply:
> mho you can add here both entries, the title can be Yarn Classic (1.0) and Yarn

+1

Reply:
@lucacavallaro I cannot add more than one value to the ring field... or can I? If I cannot add more than one value to the ring I think that the only way is to split in two cards:
1. with `ring: hold`, for `yarn 1.x`
2. with `ring: adopt` for `yarn 4.x`

Reply:
Indeed, having two distinct entries seems to be a better approach in my opinion.

Reply:
@shadowsheep1 just create two distinct entries `yarn.md` and `yarn-classic.md`, each one into the appropriate ring

Reply:
Update https://github.com/pagopa/technology-radar/pull/58/commits/fccf23ebc5fb159f1d1bb846b563302b0c7fd694